### ISSUE TITLE
Summernote react adaptation without bootstrap

### ISSUE THREAD

**Issue by erwinangeles** [summernote/react-summernote#82]
Would it be possible to compile a version of summernote.js to use the lite version? The bootstrap dependencies clash with materialize. 

As you can see summernote has a lite version without the bootstrap dependency.
https://summernote.org/getting-started/#without-bootstrap

**Comment by TheStu** [comment]
Yes this would be great. Alternatively, is there a good way to scope the bootstrap CSS to just the component? Maybe I'm just dense but it seems incredibly difficult to figure this out.

**Comment by arthedza** [comment]
Is there any update on this?

**Comment by deflexable** [comment]
hey @arthedza @erwinangeles you can try https://www.npmjs.com/package/react-summernote-lite
